packages/manager/apps/pci-savings-plan/src/pages/layout.tsx

```diff
@@ -49,7 +49,7 @@ export default function Layout() {
   } = useSavingsPlan();
 
   if (isError || isErrorSavingsPlan || error) {
-    return <ErrorBanner error={error?.response} />;
+    return <ErrorBanner error={error?.response ?? {}} />;
```

Comment:
Instead of having a simple assertion, we could enhance by using some type guard to capture the error before giving it through props

Reply:
```suggestion
    return <ErrorBanner error={error.response ?? {}} />;
```